internal/environment/environment.go

```diff
@@ -193,10 +193,6 @@ func IsDocsGeneration() bool {
 	return strings.Contains(languages, "docs")
 }
 
-func CreateGitRelease() bool {
```

Comment:
This check is pointless. It always evaluates to true. See IsLanguagePublished("swift")

Reply:
😬 thanks for cleaning this up